packages/core/src/browser/window/window-service.ts

```diff
@@ -24,6 +24,13 @@ export interface NewWindowOptions {
  * Service for opening new browser windows.
  */
 export const WindowService = Symbol('WindowService');
+
+/**
+ * Hash value that will spawb a new default window
+ */
+
+export const DEFAULT_WINDOW_HASH: string = '#!empty';
```

Comment:
typo and extra newline: 

```suggestion
/**
 * The window hash value that is used to spawn a new default window.
 */
export const DEFAULT_WINDOW_HASH: string = '#!empty';
```